frontend/src/views/GameView.vue

```diff
@@ -1,51 +1,167 @@
+						<p class="playerTwoScore"> 3 </p>
+					</div>
+
+					<canvas
+					ref="GameRef"
+					id="GameCanvas"
+					class="gameBlock"
+					width="1200"  
+					height="960">
+					</canvas>
+
+				</div>
 		</div>
-		<p class="table table-striped table-sm">
-			All Games:
-			<tbody>
-				<tr v-for="game in gameStore.allGames" :key="game.id">
-					<td> Game id: {{ game.id }} </td>
-					<td> Score player one: {{ game.score_player_one }} </td>
-					<td> Score player two: {{ game.score_player_two }} </td>
-					<td>Is active: {{ game.is_active }}</td>
-				</tr>
-				
-			</tbody>
-
-		</p>
-		<p class="table table-striped table-sm">
-			All My Games:
-			<tbody>
-				<tr v-for="game in gameStore.getMyGames" :key="game.id">
-					<td>Game id: {{ game.id }}</td>
-					<td>Score player one: {{ game.score_player_one }}</td>
-					<td>Score player two: {{ game.score_player_two }}</td>
-					<td>Is active: {{ game.is_active }}</td>
-				</tr>
-				
-			</tbody>
-			
-		</p>
 	</div>
 </template>
 
-<!--  THIS SCRIPT IS ONLY TO SHOW A WORKING GAME STORE SETUP-->
 <script lang="ts">
 import { useGameStore } from '@/stores/gameStore';
 import { defineComponent } from 'vue';
+import PlayerNames from '@/components/game-info/PlayerNames.vue'
+
+interface DataObject {
+	context: CanvasRenderingContext2D;
+}
 
 export default defineComponent({
 	name: "GameView",
+	components: {
+		PlayerNames
+	},
+
+	data(): DataObject {
+		return {
+			context: {} as CanvasRenderingContext2D,
+		}
+	},
 
 	setup(){
 		const gameStore = useGameStore();
 		gameStore.refreshData();
 		return {
 			gameStore,
 		}
-	}
+	},
+
+ 	computed: {
+		// typescript wants values used from html to have a type
+		width() {
+			return (this.$refs.GameRef as HTMLCanvasElement).width;
+		},
+		height() {
+			return (this.$refs.GameRef as HTMLCanvasElement).height;
+		},
+	},
+
+	methods: {
+		heightPercentage(percent: number): number {
+			return this.height / 100 * percent;
+		},
+		widthPercentage(percent: number) {
+			return this.width / 100 * percent;
+		},
+		
+		// https://developer.mozilla.org/en-US/docs/Web/API/CanvasRenderingContext2D/fillRect
+		drawMiddleLine() {
+			const widthStart: number = this.width / 2;
+			const heightStart: number = 0;
+			const lineWidth: number = 4;
+			const lineHeight: number = this.height;
+			
+			this.context.fillRect(
+					widthStart,
+					heightStart,
+					lineWidth,
+					lineHeight
+				);
+			},
+			
+		//  https://developer.mozilla.org/en-US/docs/Web/API/CanvasRenderingContext2D/arc
+		drawBall() {
+			const startingX: number = this.widthPercentage(40);
+			const startingY: number = this.heightPercentage(65)
+			const radius: number = 15;
+			const startAngle: number = 0;
+			const endAngle: number = Math.PI * 2; // full circle
+
+			this.context.arc(
+					startingX,
+					startingY,
+					radius,
+					startAngle,
+					endAngle,
+			)
+				// this fills the above defined arc/circle
+				this.context.fill()
```

Comment:
perhaps the ball is a little large and could be a stroke?

this.context.lineWidth=4;
this.context.stroke()

Reply:
very very nice style for the ball :D